### ISSUE TITLE
Shapely geos_c.dll not found

### ISSUE THREAD

**Issue by charlie9578** [NREL/OpenOA#142]
When trying to run OpenOA/examples/02_plant_aep_analysis.ipynb I get the following error: 

_FileNotFoundError: Could not find module 'c:\users\charlie\anaconda3\envs\openoa-env\Library\bin\geos_c.dll' (or one of its dependencies). Try using the full path with constructor syntax._

This appears to be due to Shapely and the mix of using pip and conda: https://github.com/Toblerity/Shapely/issues/1032. The solution is to run: conda install shapely

Perhaps this will be resolved though in the next release of Shapely: https://github.com/Toblerity/Shapely/pull/1108

**Comment by giswqs** [comment]
I am wondering when shapely v1.8 will be released. Shapely is a dependency of ipyleaflet. Currently, the ipyleaflet pip installation fails on Windows due to this `geos_c.dll not found` error. Would love to see this fixed. 

@davidbrochart @jorisvandenbossche @sgillies 

https://github.com/jupyter-widgets/ipyleaflet/issues/769
https://github.com/jupyter-widgets/ipyleaflet/pull/770

**Comment by jordanperr** [comment]
@charlie9578 - Thanks for pointing out this issue. We haven't run into it ourselves, but I propose to add a small note in the README with special instructions for windows users to install Shapely using conda instead of pip. Does this sound like a workable solution? Would you kindly submit a pull request into the develop branch with the change to the readme? Thank you.

**Comment by charlie9578** [comment]
Yes, that sounds good to me. Fingers crossed I've achieved a pull request!?
